#!/usr/bin/env python
PACKAGE="crc3_perception"

from dynamic_reconfigure.parameter_generator_catkin import *

gen=ParameterGenerator()

gen.add("distance_param", double_t, 0, "Until distance parameter to publish", 10.0, 1.0, 200.0)
gen.add("depth_param", double_t, 0, "Until depth", 0.1, 0.01, 0.3)
gen.add("box_param", double_t, 0, "Until sd depth", 2.5, 2, 4)
exit(gen.generate(PACKAGE,"sign_detection_node","Distance" ))
